[2022-12-04 16:41:59,730] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: first_dag.world scheduled__2022-01-24T00:00:00+00:00 [queued]>
[2022-12-04 16:41:59,929] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: first_dag.world scheduled__2022-01-24T00:00:00+00:00 [queued]>
[2022-12-04 16:41:59,938] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-12-04 16:41:59,938] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-12-04 16:41:59,940] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-12-04 16:42:00,050] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): world> on 2022-01-24 00:00:00+00:00
[2022-12-04 16:42:00,076] {standard_task_runner.py:52} INFO - Started process 901 to run task
[2022-12-04 16:42:00,113] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'first_dag', 'world', 'scheduled__2022-01-24T00:00:00+00:00', '--job-id', '156', '--raw', '--subdir', 'DAGS_FOLDER/first-dag.py', '--cfg-path', '/tmp/tmpn6lu41lj', '--error-file', '/tmp/tmpo0aiia9h']
[2022-12-04 16:42:00,131] {standard_task_runner.py:80} INFO - Job 156: Subtask world
[2022-12-04 16:42:00,517] {task_command.py:369} INFO - Running <TaskInstance: first_dag.world scheduled__2022-01-24T00:00:00+00:00 [running]> on host fb90b48d1471
[2022-12-04 16:42:13,800] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=first_dag
AIRFLOW_CTX_TASK_ID=world
AIRFLOW_CTX_EXECUTION_DATE=2022-01-24T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-24T00:00:00+00:00
[2022-12-04 16:42:13,856] {logging_mixin.py:115} INFO - Airflow
[2022-12-04 16:42:13,861] {python.py:173} INFO - Done. Returned value was: None
[2022-12-04 16:42:14,103] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=first_dag, task_id=world, execution_date=20220124T000000, start_date=20221204T164159, end_date=20221204T164214
[2022-12-04 16:42:14,517] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-12-04 16:42:16,250] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
